### PR TITLE
add bash script for data pull

### PR DESCRIPTION
**Pull request by cicorias**
bash script to retrieve bar ticker data (Date,Open,High,Low,Close,Adj Close,Volume) from Yahoo via curl.,

### CODE REVIEW

File: src/data/get_data.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
```
Comment (cicorias) [inline src/data/get_data.sh:2]: I’d like to make this all python... 

datetime.datetime(2012,4,1,0,0).timestamp()
1333234800.0

**Comment by tedsmoore** [comment]
Ok this is starting to clean up. I went ahead and made a couple changes per our slack convo.

1) I moved the data dump up one level to a "data" folder at the root level. This way we can easily access it from both notebooks and our modules (and keeps raw data out of src). Then I'm thinking that in our Google Drive folder, we can make 2 directories (notebooks, data) that mimic this layout so that notebook code works in both locations.
2) Added the new data folder to .gitignore

Let me know if you had a different structure in place, just a suggestion.

**Review by tedsmoore — APPROVED** [review]
Went ahead and made the changes outlined in the comment. Will wait to merge until you see those. Feel free to tweak.

**Review by cicorias — COMMENTED** [review]
LGTM

**Pull request merged**